docs/contributor/04-local-test-setup.md

```diff
@@ -104,8 +104,7 @@ This setup is deployed with the following security features enabled:
    make build-manifests
    
    # create the a ModuleTemplate CR and save it to the template.yaml file
-   kyma alpha create module -p ./ --version 1.2.3 \
-   --registry k3d-registry.localhost:5111 --insecure --module-config-file ./module-config.yaml
+   modulectl create --module-config-file ./module-config.yaml --registry http://k3d-registry.localhost:5111 --insecure 
```

Comment:
If the PR for shortening the config file flag gets merged first, please make sure to update the flag here

Reply:
We are using released version here. Changes to the modulectl won't affect KLM e2e until we release a new version and use it the modulectl action.